.github/workflows/test-cli-js.yml

```diff
@@ -39,9 +39,9 @@ jobs:
         uses: dtolnay/rust-toolchain@stable
 
       - name: setup node
-        uses: actions/setup-node@v2
+        uses: actions/setup-node@v4
         with:
-          node-version: 18
+          node-version: 20
```

Comment:
Please adjust this to node@18. We would like to run the tests on the oldest LTS/maintenance release. That being said, if there is an alias for it, it would be nice to not have to change it.